GameOfLife.java

```diff
@@ -84,22 +128,56 @@ private static int[][] evolve(int[][] board) {
 	// Assumes that i is at least 1 and at most the number of rows in the board - 1. 
 	// Assumes that j is at least 1 and at most the number of columns in the board - 1. 
 	// Uses the count(board,i,j) function to count the number of alive neighbors.
-	private static int cellValue(int[][] board, int i, int j) {
+	public static int cellValue(int[][] board, int i, int j) {
+		int countNeighbors=count(board, i,  j);
+		int zeroOrOne =board[i-1][j-1];
+		if(board[i-1][j-1]==1){
+			if(countNeighbors==1){
+				zeroOrOne = 0;
+			}
+			else if(countNeighbors>3){
+				zeroOrOne = 0;
+			}
```

Comment:
Let's merge the conditions
```suggestion
			if ((countNeighbors==1) || (countNeighbors>3)) {
				zeroOrOne = 0;
			}
```